lib/say_when/storage/active_record_strategy.rb

```diff
@@ -71,24 +71,25 @@ def self.acquire_next(no_later_than = nil)
           check_connection
           hide_logging do
             SayWhen::Storage::ActiveRecordStrategy::Job.transaction do
-              # select and lock the next job that needs executin' (status waiting, and after no_later_than)
```

Comment:
Not a fan of folksy comments, huh?

Reply:
I probably was in a mood, but looks like a mistake or open single quote, so changed it